packages/@lwc/synthetic-shadow/src/polyfills/iframe-content-window/patch.ts

```diff
@@ -5,8 +5,11 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
 import { getOwnPropertyDescriptor, defineProperty, isNull, isUndefined } from '@lwc/shared';
+import { partialStructuredClone } from '@locker/near-membrane-shared';
```

Comment:
I still don't understand what the problem is, so this looks very suspicious.

Reply:
When objects created in a realm leave the realm they are represented as a near-membrane proxy. When that proxy hits `iframe.contentWindow.postMessage` it will fail to be structured cloned and error.

Reply:
LWS will distort iframe and window of the code in the realm. However the `iframe.contentWindow.postMessage` called from system mode code (that outside of a realm) cannot be patched by LWS. The issue is that when a value leaves a realm it is represented as a near-membrane proxy. LWS cannot augment an `iframe.contentWindow` referenced by other system mode code as in synthetic-shadow because there is no `Window.prototype` that covers the bolt on method `postMessage`. So every iframe in system mode has this gotcha. LWS can patch the primary `window.postMessage` but does not reach iframes.